backend/infrahub/git/tasks.py

```diff
@@ -344,13 +347,8 @@ async def pull_read_only(model: GitRepositoryPullReadOnly) -> None:
 @flow(name="git-repository-merge")
 async def merge_git_repository(model: GitRepositoryMerge) -> None:
     service = services.service
-    log.info(
-        "Merging repository branch",
-        repository_name=model.repository_name,
-        repository_id=model.repository_id,
-        source_branch=model.source_branch,
-        destination_branch=model.destination_branch,
-    )
+    await add_branch_tag(branch_name=model.source_branch)
+    await add_branch_tag(branch_name=model.destination_branch)
```

Comment:
Not a blocker but here it looks like we should have a function that can take two branch_names (or tags) as input and send one request with both of them as it each call to the add_branch_tag will issue an HTTP request.

Reply:
Agreed, we should probably also merge that with `related_node`, I'll do that in another PR